### DISCUSSION TITLE
Party Frames?

### DISCUSSION THREAD

**Discussion by ArchfiendNZ** [tirem/XIUI#4] [Ideas]
Hello there,

This is my favorite ashita4 mod, thank you so much for making it.

There is only one thing missing that would make it absolutely perfect which is party frames, there are no party frames currently available for Ashita4 that are maintained/fleshed out. If you added this feature that would be absolutely amazing.

Keep up the fantastic work.

Cheers,
Archfiend (HorizonXI)

**Comment by tirem** [comment]
Please look forward to it ;)

I'm currently working on it actually. That and continued debugging of course as well as a configuration UI for scaling, offsets, colors, etc.

**Comment by tirem** [comment]
Added! Buffs and debuffs to come soon!

**Comment by ArchfiendNZ** [comment]
Looks great, size is a bit big for me but maybe I can find out how to scale it in the meantime :)

Wicked work

**Comment by ArchfiendNZ** [comment]
Got one bug where instead of highlighting the party member, the highlight covers three party members, assume its stretching for some reason

Let me know if you want me to create new post in the Issues section

![image](https://user-images.githubusercontent.com/93624292/211191543-37bc8d22-bd8f-4ae3-a0b8-e09e96d83453.png)

**Comment by tirem** [comment]
Built it on an ultrawide monitor, different resolutions are getting weird results with the selector. Working on it today. There is a bunch of options in the settings.lua file but there will be a simpler config UI soon :)

**Comment by ArchfiendNZ** [comment]
Thank you. I am using 51:9 so I will look at settings.lua and see if I can tweak it.

**Comment by ArchfiendNZ** [comment]
Working perfect with new version and the scaling down across the board has helped a hell of a lot.

Thanks!!!

**Comment by Watercrown** [comment]
Such a great addon, thank you so much!

Is there any chance we will get a solid background for the party frame? I would really like to obscure the default one.
